extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -694,4 +719,100 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 }
 
+/**
+ * Keeps track of the third-party vendors' cross-domain iframes. Only one
+ * static instance will be created, which will keep manage ALL amp-analytics
+ * tags.
+ */
+class AmpAnalyticsFrames {
+  constructor() {
+    this.map = new Map();
```

Comment:
Name something more descriptive than 'map'. What is the purpose of this map? Could the various properties stored within all instead be instance variables on the class? It seems like the map is a layer of obfuscation that will make reading this class less clear in the future (unless there is a reason for the map I don't know).